stable_baselines3/common/type_aliases.py

```diff
@@ -56,7 +56,7 @@ class DictReplayBufferSamples(ReplayBufferSamples):
 
 
 class RolloutReturn(NamedTuple):
-    episode_reward: float
+    episode_reward: float  # not used anymore, kept for backward compat
```

Comment:
Is this required by any of our code? I think it would be clearer to outright remove it not to introduce hidden bugs for those who use this variable.

Reply:
I was mainly thinking of people who created custom algorithms, this variable is only meant for logging...
But you are probably right, we should probably just get rid of it.